Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [workspace]
 resolver = "2"
 members = [
-  "compiler-cli",
-  "compiler-core",
-  "compiler-wasm",
-  "test-helpers-rs",
-  "test-package-compiler",
-  "test-project-compiler",
+    "compiler-cli",
+    "compiler-core",
+    "compiler-wasm",
+    "test-helpers-rs",
+    "test-package-compiler",
+    "test-project-compiler",
+    "test-echo-output",
```

Comment:
Rather than making this specific to echo let's make this a set of tests related to what running a program prints.